src/LAND/LAND.router.ts

```diff
@@ -0,0 +1,194 @@
+import { env } from 'decentraland-commons'
+import { server } from 'decentraland-server'
+import FormData from 'form-data'
+import { Request } from 'express'
+//@ts-ignore
+import * as contentHash from 'content-hash'
+import fetch, { Response as FetchResponse } from 'node-fetch'
+import { HTTPError, STATUS_CODES } from '../common/HTTPError'
+import { Router } from '../common/Router'
+import { getCID } from '../utils/cid'
+import {
+  GetRedirectionHashesResponse,
+  UploadRedirectionResponse,
+} from './LAND.types'
+
+export class LANDRouter extends Router {
+  mount() {
+    this.router.get(
+      '/lands/redirection/hashes',
```

Comment:
I was thinking that we could rename this endpoint to:
```suggestion
      '/lands/redirection-hashes',
```
To be a bit more RESTfullish.

Reply:
Like the idea.
I would do it camel case like the /assetPacks endpoint

Reply:
Renamed to /redirectionHashes